<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNoise</class>
 <widget class="QWidget" name="addNoise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Noise</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Noise</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="train">
        <property name="text">
         <string>Training Set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="test">
        <property name="text">
         <string>Testing Set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="both">
        <property name="text">
         <string>Both</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="neither">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Neither</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Example Image From Dataset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply noise to:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="confirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>158</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Selection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continue1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(Note the displayed image may not be to actual scale</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="noiseSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>15</width>
     <height>149</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>